src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: A modern web experience on Adobe Experience Manager with WorkBox
+description: >
+  Progressive Web Apps leverage what the modern web can do. Adobe used Workbox to bring those capabilities to Adobe Experience Manager.
+authors:
+  - joemedley
+date: 2022-03-04
+hero: image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/RKF1rl4ItVpoMj7TRZHG.png
```

Comment:
Per the [images and video](https://web.dev/handbook/markup-media/) guidelines, hero images must be 3200px wide by 960px.

Reply:
I uploaded a new one.